src/platform-includes/distributed-tracing/custom-instrumentation/dotnet.mdx

```diff
@@ -0,0 +1,64 @@
+On this page you will learn how to manually propagate trace information into and out of your .NET application. Please note, that you do not need to do this manually, if you [use one of our supported frameworks](/platforms/dotnet/usage/distributed-tracing/#how-to-use-distributed-tracing), or you have our <PlatformLink to="/performance/">performance monitoring feature</PlatformLink> turned on.
+
+To set it up manually, all you have to do is to make sure your application extracts incoming headers and to set those headers again when making an outgoing request within your application.
+
+## Step 1) Extract Incoming Tracing Information
+
+Incoming tracing information has to be extracted and stored in memory for later use. Sentry provides the `ContinueTrace()` function to help you with this. Incoming tracing information can come from HTTP headers, for example, by another Sentry SDK used in your frontend project.
+
+Here's an example of how to extract and store incoming tracing information using `ContinueTrace()`:
+
+```csharp
+using System;
+using System.Net.Http;
+using Sentry;
+
+var httpContext = ...;
+
+var sentryTraceHeader = httpContext.Request.Headers.GetValueOrDefault("sentry-trace");
+var sentryBaggageHeader = httpContext.Request.Headers.GetValueOrDefault("baggage");
```

Comment:
As we talked offline lets not point users to HTTP here, we auto instrument ASP.NET and ASP.NET Core so they can get it from `SentrySdk` or start a new transaction and bind that to the scope and then it's avaiable.